bcgov/GDX-Analytics
# updated lock file and bumped tar-6.2.1 & express-4.19.12

This PR updates the package-lock.json with a fresh npm install and bumps tar-6.2.1 & express-4.19.12, solving the remaining dependabot alerts for the examples/vue_js_tracker.
Testing instructions:
- clone the repo and switch to branch
```
git clone https://github.com/bcgov/GDX-Analytics.git
cd GDX-Analytics/
git checkout GDXDSD-6852-vuejs-tracker-branches
```
- install vue and template compiler globally
```
npm install -g @vue/cli
npm i -g vue-template-compiler
```
- install the dependencies listed in the package-lock.json
```
cd examples/vue_js_tracker/
npm ci
```
- start a development server
```
npm run serve
```
check if sample vue.js page is running by browsing to http://localhost:8080/
